Formula/allureofthestars.rb

```diff
@@ -7,7 +7,7 @@ class Allureofthestars < Formula
   homepage "https://www.allureofthestars.com/"
   url "https://hackage.haskell.org/package/Allure-0.9.5.0/Allure-0.9.5.0.tar.gz"
   sha256 "8180fe070633bfa5515de8f7443421044e7ad4ee050f0a92c048cec5f2c88132"
-  license "AGPL-3.0"
+  license "AGPL-3.0-or-later"
```

Comment:
Where do you see an indicator of `-or-later`. I think it should be `AGPL-3.0-only`.

Reply:
the self-declared license is in here, https://hackage.haskell.org/package/Allure-0.9.5.0

Reply:
![image](https://user-images.githubusercontent.com/1580956/89962849-ed2f2280-dc13-11ea-80a5-3493f659aad4.png)

Reply:
This is actually the strategy I took to iterate thru all these formulae (rather going thru one by one, I just cross-checked the haskell.org's index.)

Reply:
Sounds good to me